rate-providers/yUSDRateProviderArbitrum.md

```diff
@@ -0,0 +1,53 @@
+# Rate Provider: `ChainlinkRateProvider`
+
+## Details
+- Reviewed by: @brunoguerios
+- Checked by: @\<GitHub handle of secondary reviewer\>
+- Deployed at:
+    - [arbitrum:0xb1D5c23DbfCb36864f32bc704AE4205BB9720C33](https://arbiscan.io/address/0xb1D5c23DbfCb36864f32bc704AE4205BB9720C33)
+- Audit report(s):
+    - [Protocol Audits](https://docs.yield.fi/resources/audits)
+
+## Context
+This RateProvider is updated via the API3 product. It reports the exchangeRate of yUSD/USD on mainnet. It is read on mainnet and available as an Oracle on arbitrum.
+
+## Review Checklist: Bare Minimum Compatibility
+Each of the items below represents an absolute requirement for the Rate Provider. If any of these is unchecked, the Rate Provider is unfit to use.
+
+- [x] Implements the [`IRateProvider`](https://github.com/balancer/balancer-v2-monorepo/blob/bc3b3fee6e13e01d2efe610ed8118fdb74dfc1f2/pkg/interfaces/contracts/pool-utils/IRateProvider.sol) interface.
+- [x] `getRate` returns an 18-decimal fixed point number (i.e., 1 == 1e18) regardless of underlying token decimals.
+
+## Review Checklist: Common Findings
+Each of the items below represents a common red flag found in Rate Provider contracts.
+
+If none of these is checked, then this might be a pretty great Rate Provider! If any of these is checked, we must thoroughly elaborate on the conditions that lead to the potential issue. Decision points are not binary; a Rate Provider can be safe despite these boxes being checked. A check simply indicates that thorough vetting is required in a specific area, and this vetting should be used to inform a holistic analysis of the Rate Provider.
+
+### Administrative Privileges
+- [ ] The Rate Provider is upgradeable (e.g., via a proxy architecture or an `onlyOwner` function that updates the price source address).
+
+- [x] Some other portion of the price pipeline is upgradeable (e.g., the token itself, an oracle, or some piece of a larger system that tracks the price). \<Delete this hint: If unchecked, delete all of the bullets below.\>
+    - upgradeable component: `Api3ReaderProxyV1` ([arbitrum:0xE88DA976479461080072D6461128fd401B6D4Dcb](https://arbiscan.io/address/0xE88DA976479461080072D6461128fd401B6D4Dcb#readProxyContract))
+    - admin address: [arbitrum:0x2AAE699ed04BBbD068f67a5b3C813eBB35f2c9E8](https://arbiscan.io/address/0x2AAE699ed04BBbD068f67a5b3C813eBB35f2c9E8)
+    - admin type: multisig
+        - multisig threshold/signers: 4/8
+        - multisig timelock? NO
```

Comment:
I couldn't find timelocks in the implementation, but if it's too relevant, it would be nice if you could double check.

Reply:
Not every system has them. They are helpful for the reader to have but not a requirement.